app/helpers/ink_components/form_builder.rb

```diff
@@ -68,12 +68,13 @@ def select(attribute, choices = nil, select_options = {}, tag_options = {}, &)
     end
 
     def file_field(attribute, **opts)
+      self.multipart = true
       file_input_component(**html_options(attribute, objectify_options(opts)), **sanitize_options(opts))
     end
 
     def text_area(attribute, **options)
```

Comment:
Como os outros seguem o padrão de opts, é melhor continuar seguindo.

```suggestion
    def text_area(attribute, **opts)
```